digiwf-task/digiwf-tasklist-service/src/main/java/de/muenchen/oss/digiwf/task/service/application/port/out/polyflow/TaskNotFoundException.java

```diff
@@ -1,8 +1,12 @@
 package de.muenchen.oss.digiwf.task.service.application.port.out.polyflow;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
 /**
  * Signals that the task with given id is not available.
  */
+@ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
Hätte nicht nur as gereicht?
Jetzt ist halt das Error-Handling nur für den einen Fall über den separaten Error.

Reply:
Nee. weil im Frontend muss da bzgl. axios generell das Errorhandling umgebaut werden.

Reply:
Gibt es da dann schon ein Nachfolge-Issue dazu? Sollte man sonst vll gleich anlegen und verlinken, dass das nicht unter geht

Reply:
Ich sammle aktuell schon die ganzen technical deps im Frontend und gieße das in Tickets. 
Ist mein Ziel am Freitag im Refinement vorzustellen

Reply:
Refactoring von Frontend wurde festgehalten in: https://github.com/it-at-m/digiwf-core/issues/1088